core/src/commonMain/kotlin/com/ustadmobile/core/viewmodel/HolidayCalendarDetailViewModel.kt

```diff
@@ -0,0 +1,8 @@
+package com.ustadmobile.core.viewmodel
+
+import com.ustadmobile.lib.db.entities.HolidayCalendar
+
+data class HolidayCalendarDetailUIState(
+    val holidayCalendar: HolidayCalendar? = null,
+    val calendarList: List<HolidayCalendar>? = emptyList()
```

Comment:
The HolidayCalendar is basically a name linked to a list of Holidays. Each Holiday has a holiday start time and end time (e.g. Eid, summer vacation, winter vacation, etc).

For example there could be a HolidayCalendar for the cold area schedule schools and warm area schedule schools.

e.g. 
```
val holidayCalendar: HolidayCalendar? = null
val calendarList: List<Holiday> = emptyList()
```